run.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Create_kafka test
+  hosts: localhost
+  gather_facts: true
+  connection: local
+  tasks:
+  - name: Print the env var value
+    debug:
+      msg: "{{ lookup('env', 'OFFLINE_TOKEN') }}"
+  - name: Create kafka
+    create_kafka:
+      name: "struttin"
+      region: "us-east-1"
+    register:
+      kafka_req_resp 
+  - name: Debug create kafka
+    debug:
+      msg: "{{ kafka_req_resp }}"
+  - name: Create Service Account
+    create_service_account:
+      name: "struttin-jeeves"
+      description: "struttin-jeeves service account"
+    register:
+      srvce_acc_resp_obj
+  - name: Debug service account
+    debug:
+      msg: "{{ srvce_acc_resp_obj }}"
+  # - name: Create kafka ACL Service Binding
```

Comment:
? Service binding is different story. We attach ACL to service accounts

Reply:
that is essentially the test script that will form the basis for the `create and configure kafka` (may need to come up with a better name) role.

I will remove that once the role is created